### PR TITLE
Add container name and tenant id for linux consumption logs

### PR DESCRIPTION
**Pull request by divyagandhii**
Emit container name and tenant id when emitting functions logs from linux container

### CODE REVIEW

File: src/WebJobs.Script.WebHost/Diagnostics/LinuxContainerEventGenerator.cs [hunk 1]
```diff
@@ -66,6 +73,24 @@ private void ConsoleWriter(string evt)
...
         }
 
+        private void TryReadEnvironmentVariables()
+        {
+            if (string.IsNullOrEmpty(_stampName))
+            {
+                _stampName = _environment.GetEnvironmentVariable("WEBSITE_HOME_STAMPNAME")?.ToLowerInvariant();
+            }
+
+            if (string.IsNullOrEmpty(_containerName))
```
Comment (balag0) [inline src/WebJobs.Script.WebHost/Diagnostics/LinuxContainerEventGenerator.cs]: You can cache the containername in constructor it wont change ever and will be part of initial settings.

File: src/WebJobs.Script.WebHost/Diagnostics/LinuxContainerEventGenerator.cs [hunk 2]
```diff
@@ -66,6 +73,24 @@ private void ConsoleWriter(string evt)
...
+            {
+                _containerName = _environment.GetEnvironmentVariable("CONTAINER_NAME")?.ToUpperInvariant();
+            }
+
+            if (string.IsNullOrEmpty(_tenantId))
```
Comment (balag0) [inline src/WebJobs.Script.WebHost/Diagnostics/LinuxContainerEventGenerator.cs]: Can you add getters for tenantid and stampname and move this logic there.

File: src/WebJobs.Script.WebHost/Diagnostics/LinuxContainerEventGenerator.cs [hunk 3]
```diff
@@ -10,39 +10,68 @@ internal class LinuxContainerEventGenerator : LinuxEventGenerator
...
+        public LinuxContainerEventGenerator(IEnvironment environment, Action<string> writeEvent = null)
         {
             _writeEvent = writeEvent ?? ConsoleWriter;
-
+            _environment = environment;
             if (Environment.GetEnvironmentVariable(EnvironmentSettingNames.ConsoleLoggingDisabled) == "1")
             {
                 _consoleEnabled = false;
             }
+            _containerName = _environment.GetEnvironmentVariable("CONTAINER_NAME")?.ToUpperInvariant();
```
Comment (mathewc) [inline src/WebJobs.Script.WebHost/Diagnostics/LinuxContainerEventGenerator.cs]: nit: Use EnvironmentSettingNames.ContainerName

File: src/WebJobs.Script.WebHost/Diagnostics/LinuxContainerEventGenerator.cs [hunk 4]
```diff
@@ -10,39 +10,68 @@ internal class LinuxContainerEventGenerator : LinuxEventGenerator
...
 
         public static string DetailsEventRegex { get; } = $"{ScriptConstants.LinuxFunctionDetailsEventStreamName} (?<AppName>[^,]*),(?<FunctionName>[^,]*),\\\\\"(?<InputBindings>.*)\\\\\",\\\\\"(?<OutputBindings>.*)\\\\\",(?<ScriptType>[^,]*),(?<IsDisabled>[0|1])";
 
+        private string StampName
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(_stampName))
+                {
+                    _stampName = _environment.GetEnvironmentVariable("WEBSITE_HOME_STAMPNAME")?.ToLowerInvariant();
```
Comment (mathewc) [inline src/WebJobs.Script.WebHost/Diagnostics/LinuxContainerEventGenerator.cs]: Add these new ones to EnvironmentSettingNames?

**Pull request merged**